exercises/simple-linked-list/linked_list.exs

```diff
@@ -0,0 +1,73 @@
+defmodule LinkedList do
+  @opaque t :: any()
+
+  @doc """
+  Construct a new LinkedList
```

Comment:
I think using the module name is a bit confusing in this and the following comments. Perhaps, 

"Construct an empty linked list"